guest/bin/internal-makepkg

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
-export HOME="/home/$(whoami)"
+set -euo pipefail
+
+export HOME="/home/main-builder"
 export LANG='en_US.UTF-8'
-export USER="$(whoami)"
+export USER="main-builder"
 export XDG_RUNTIME_DIR='/tmp'
 export SHELL='/bin/bash'
 export TERM='dumb'
 
-cd ~/pkgwork || exit
-exec makepkg $@
+chown -R "$USER":"$USER" "${HOME}"
+
```

Comment:
I agree with all changes right now, but I need to test how the ones in this file will affect the builds using nspawn...

Reply:
good luck